perf/reader.go

```diff
@@ -162,6 +163,7 @@ type ReaderOptions struct {
 	// Read will process data. Must be smaller than PerCPUBuffer.
 	// The default is to start processing as soon as data is available.
 	Watermark int
+	TimeOut   int
```

Comment:
I think this option should have at least document what it is used for so users can reason about its impact. Also if there is a default value, it should be mentioned here as well I think.

Reply:
Ok, thanks

Reply:
With `SetDeadline` this Option isn't used anymore.